RecoBTag/FeatureTools/test/u0_cfg.py

```diff
@@ -0,0 +1,74 @@
+# Unit test configuration file for RecoBTagInfo producers:
+# Verify the use of unsubtracted jet map
+#
+
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("TEST")
+
+process.load('Configuration.Geometry.GeometryDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_38T_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+process.load("TrackingTools.TransientTrack.TransientTrackBuilder_cfi")
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(2)
+)
+
+process.source = cms.Source("PoolSource",
+        fileNames = cms.untracked.vstring("root://xrootd-cms.infn.it//store/hidata/HIRun2023A/HIPhysicsRawPrime0/MINIAOD/PromptReco-v2/000/375/823/00000/8158260e-df3c-45a5-a121-55345a682a23.root")
```

Comment:
@stahlleiton , please can you update this to use logical file name (lfn) e.g. `/store/hidata/HIRun2023A/HIPhysicsRawPrime0/MINIAOD/PromptReco-v2/000/375/823/00000/8158260e-df3c-45a5-a121-55345a682a23.root` so that this file can be picked up from bot eos cache instead of accessing it from `infn`. This unit test is failing in IBs as the file is not accessible

Reply:
Done in https://github.com/cms-sw/cmssw/pull/44522